samples/trace-utility-network/README.md

```diff
@@ -55,7 +55,7 @@ The [Naperville electrical](https://sampleserver7.arcgisonline.com/server/rest/s
 
 Using utility network on ArcGIS Enterprise 10.8 requires an ArcGIS Enterprise member account licensed with the [Utility Network user type extension](https://enterprise.arcgis.com/en/portal/latest/administer/windows/license-user-type-extensions.htm#ESRI_SECTION1_41D78AD9691B42E0A8C227C113C0C0BF). Please refer to the [utility network services documentation](https://enterprise.arcgis.com/en/server/latest/publish-services/windows/utility-network-services.htm).
 
-A `UtilityNetworkTrace` toolkit component can be used for various utility network related use cases. For information about setting up the toolkit, as well as code for the underlying component, visit the [toolkit repository](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit).
+A `UtilityNetworkTrace` toolkit component can be used for various utility network related use cases. For information about setting up the toolkit, as well as code for the underlying component, visit the [toolkit repository](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit).
```

Comment:
```suggestion
```

Not sure how this got in, but the trace-utility-network sample does NOT use a toolkit component as it only showcases trace to visualize and validate the network topology of a utility network. This sample doc should be removed @rom14514.